<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultPage</class>
 <widget class="QMainWindow" name="ResultPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result Page</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="DiseaseList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>256</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="DrugList">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>271</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="SideEffectList">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>531</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 걸린 질병 목록</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>추천 약 조합</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>부작용</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>취약 질병 목록</string>
    </property>
   </widget>
   <widget class="QListWidget" name="RelatedDiseaseList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>256</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
